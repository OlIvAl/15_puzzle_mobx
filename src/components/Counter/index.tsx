 import React from 'react';

interface IProps {
  count: number
}

const Counter: React.FC<IProps> = ({count}) => (
  <div><strong>Count:</strong> {count}</div>
);

export default Counter