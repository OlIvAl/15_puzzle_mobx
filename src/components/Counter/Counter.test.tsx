import * as React from 'react';
import Counter from '.';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Timer', () => {
  const fakeCounter: number = 11;

  it('он отображается', () => {
    const wrapper = shallow(<Counter count={fakeCounter} />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('отображает счетчик', () => {
    const wrapper = shallow(<Counter count={fakeCounter} />);

    expect(wrapper.contains(fakeCounter)).toEqual(true);
  });
});