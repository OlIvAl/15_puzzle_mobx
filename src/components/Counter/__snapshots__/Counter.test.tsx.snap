// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer он отображается 1`] = `
<div>
  <strong>
    Count:
  </strong>
   
  11
</div>
`;
